packages/react-data-grid/src/Cell/CellValue.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { isElement, isValidElementType } from 'react-is';
+
+import { FormatterProps } from '../common/types';
+import { SimpleCellFormatter } from '../formatters';
+import { CellContentProps } from './CellContent';
+
+type CellValueProps = Pick<CellContentProps,
+'rowIdx'
+| 'rowData'
+| 'column'
+| 'value'
+| 'isScrolling'
+>;
+
+export default function CellValue({ rowIdx, rowData, column, value, isScrolling }: CellValueProps) {
+  function getRowData() {
```

Comment:
`getRowData` is called within `getFormatterProps` and `getFormatterDependencies`, let's just inline it in `getFormatterProps`, and pass the data to `getFormatterDependencies`, so we don't do the same work twice.

Reply:
Done